Nasal/light-manager.nas

```diff
@@ -0,0 +1,303 @@
+# provides relative vectors from eye-point to aircraft lights
+# in east/north/up coordinates the renderer uses
+
+var light_manager = {
+    run: 0,
+
+    lat_to_m: 110952.0,
+    lon_to_m: 0.0,
+
+    light1_xpos: 0.0,
```

Comment:
Yes, you can but I am at a loss as to why? I tested it and you can "only" create the me.var in the init section, and it works (Lua style). But why do we want to do this? It would then make it difficult at best to quickly see what a light_manager object consists of and what are we really gaining?